React/Base/RCTBridge.m

```diff
@@ -1052,6 +1054,17 @@ - (void)initJS
         [[NSNotificationCenter defaultCenter] postNotificationName:RCTJavaScriptDidFailToLoadNotification
                                                             object:_parentBridge
                                                           userInfo:userInfo];
+        
+        /**
+         * Register the display link to start sending js calls after everything
+         * is setup
+         */
+        NSRunLoop *targetRunLoop = [_javaScriptExecutor isKindOfClass:[RCTContextExecutor class]] ? [NSRunLoop currentRunLoop] : [NSRunLoop mainRunLoop];
+        [_jsDisplayLink addToRunLoop:targetRunLoop forMode:NSRunLoopCommonModes];
+        
+        [[NSNotificationCenter defaultCenter] postNotificationName:RCTJavaScriptDidLoadNotification
+                                                            object:_parentBridge
+                                                          userInfo:@{ @"bridge": self }];
```

Comment:
I don't think we need it here. If there was an error, there's no JavaScript running, so we don't need the display link notifications. We also can't send a notification that JavaScript has been loaded, since it wasn't.